QUARKUS-2489.md

```diff
@@ -0,0 +1,36 @@
+# QUARKUS-2489 OIDC Back channel logout
+
+Jira: https://issues.redhat.com/browse/QUARKUS-2489
+
+This feature provides Back-Channel Logout capability for OIDC extension.
+
+## Scope of the testing
+
+Backchannel logout applies to `quarkus-oidc` and basically is a standard way to logout the current user from all the applications this user is currently logged in, bypassing the user agent.
+
+We will create a valid token by an authorization flow and proceed with the logout flow.
+### Impact on test suites and testing automation
+
+A New logout endpoint is going to be created in `keycloak-oidc-client-extended` and `keycloak-oidc-client-reactive-extended`, and we will need to cover JVM and native mode in order to double-check that there are no conflicts with the native compilations. 
```

Comment:
> A New logout endpoint is going to be created in `keycloak-oidc-client-extended` and `keycloak-oidc-client-reactive-extended`

Can you please explain (in comment is enough) why would you create new logout endpoint? AFAICT Quarkus has handler that expects logout token on the logout back-channel path , so there are 2 possible situations:

- you call RP that is also OP (KC?), and KC calls the tested app (RP)
- back-channel logout in the tested app (Quarkus provides route handler itself)

In neither of these scenarios, you need the logout endpoint. Maybe you mean some endpoint that checks you are logged out (if we don't have such endpoint? please check)?

Reply:
You could add a `post-logout` endpoint, let's make it the most simple as we can, and maybe in the future, we can make a more advance scenario. So +1 to remove it!.